language/tools/move-cli/src/lib.rs

```diff
@@ -236,26 +240,42 @@ impl OnDiskStateView {
             .collect()
     }
 
-    pub fn view_module(&self, module_path: &Path) -> Result<Option<String>> {
+    fn view_bytecode(path: &Path, is_module: bool) -> Result<Option<String>> {
```

Comment:
There really isn't a ton of shared code between these two versions. Maybe best to just inline it into each version?